XP-Pen_Artist_16_Pro/index.md

```diff
@@ -0,0 +1,19 @@
+---
+VID: "28bd"
+PID: "900a"
+vendor: XP-Pen
+product: Artist 16 Pro
+image: XP-Pen_Artist_16_Pro
+report_rate: 266
+working_area:
+    width: 16
+    height: 9
+    resolution: 1080
+pen:
+    pressure_levels: 8192
+frame_controls: 8 buttons
+sold_as:
+    - XP-Pen Artist 16 Pro
```

Comment:
This just repeats the vendor/product entry. I expect one of them should be without the "Pro".